### PR TITLE
[WIP] Developer bundle

### PR DESCRIPTION
**Pull request by dantleech**
This PR adds a developer bundle.

The developer bundle contains development fixtures for the entire Sulu stack, it should only be enabled when required.

This is a WIP until the search mapping PR is merged.

### CODE REVIEW

File: src/Sulu/Bundle/DeveloperBundle/DataFixtures/ORM/LoadContactData.php [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+    {
+        $faker = Faker\Factory::create('en_GB');
+
+        $account = new Account();
+        $account->setLft(0);
+        $account->setRgt(1);
+        $account->setDepth(0);
+        $account->setName('Musterfirma');
+        $account->setCreated(new \DateTime());
+        $account->setChanged(new \DateTime());
```
Comment (dantleech) [inline src/Sulu/Bundle/DeveloperBundle/DataFixtures/ORM/LoadContactData.php]: Not used.

File: src/Sulu/Bundle/DeveloperBundle/DataFixtures/ORM/LoadContactData.php [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+
+        return $address;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
+    public function getOrder()
+    {
+        return 100;
```
Comment (dantleech) [inline src/Sulu/Bundle/DeveloperBundle/DataFixtures/ORM/LoadContactData.php]: Have to load after the other fixtures as they all hack the ID to 1 - the configuration references the ID directly .. :(

**Comment by chirimoya** [comment]
@dantleech @danrot ... would it make sense to move this to https://github.com/sulu-io/SuluDeveloperBundle and close this PR?

**Comment by danrot** [comment]
I would say so. The DeveloperBundle should IMHO not be part of the standard edition.

**Comment by dantleech** [comment]
Yeah, this is very outdated. The DeveloperBundle already has contact fixutres I think.

**Pull request closed without merging**